compiler/src/dotty/tools/MainGenericRunner.scala

```diff
@@ -137,6 +141,19 @@ object MainGenericRunner {
         val newSettings = if arg.startsWith("-") then settings else settings.withPossibleEntryPaths(arg).withModeShouldBePossibleRun
         process(tail, newSettings.withResidualArgs(arg))
 
+  // copy of method private to dotty.tools.dotc.config.CliCommand.distill()
+  // TODO: make it available as a public method and remove this copy?
```

Comment:
Yes, please avoid copying code if it can be refactored inside.

Reply:
Should I be able to import a method from a trait?   It complains when I try to do so.

If I refactor the `def expandArg` to somewhere outside of `trait CliCommand`, where would a good place be?

Reply:
> Should I be able to import a method from a trait? It complains when I try to do so.

You could create an object CliCommand in the same file as the trait and put the method there, but CommandLineParser would be fine too.